### ISSUE TITLE
New deposits count smaller than total deposits count

### ISSUE THREAD

**Issue by JustinDrake** [sigp/lighthouse#2006]
A fresh beacon node printed the following log line. Notice that the number of new deposits (25497) is greater than the number of total deposits (25484) which may be a bit confusing for node operators.

```
Nov 28 17:05:18.193 INFO Imported deposit log(s)                 new: 25497, total: 25484, latest_block: 11346526, service: eth1_rpc
```

**Comment by pawanjay176** [comment]
@JustinDrake I suspect this is happening because of duplicate logs that we get when adding logs to our deposit cache.
we don't add the log to the deposit cache if it's a duplicate but we still return a success(which increases the newly imported count) as long as the log is valid.

**Comment by pawanjay176** [comment]
Fixed in #2038
